legacy/app/data/common/post-edit-create/media-edit.service.js

```diff
@@ -62,6 +62,12 @@ function (
             });
             return deferred.promise;
         },
+        confirmDelete: function(mediaId) {
```

Comment:
There is already a confirmDelete function in the Notify-service you can use. By using that, you will get a dialog with styles that are adjusted to the UI. It is situated here: https://github.com/ushahidi/platform-client/blob/develop/legacy/app/common/notifications/notify.service.js#L287 and you can see how it is used for example here: https://github.com/ushahidi/platform-client/blob/develop/legacy/app/settings/categories/categories.controller.js#L44

Reply:
alright thanks, let me  check out that